### PR TITLE
Import/Export integration

### PR DESCRIPTION
**Pull request by seemueller**

### CODE REVIEW

File: fusionbackend/src/main/java/io/fusion/fusionbackend/rest/fleetmanager/FleetAssetRestService.java [hunk 1]
```diff
@@ -74,6 +82,16 @@ public AssetDto getAsset(@PathVariable final Long companyId,
                 embedChildren);
     }
 
+    @GetMapping(path = "/companies/{companyId}/assetseries/{assetSeriesId}/assets/{assetId}/export")
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/rest/fleetmanager/FleetAssetRestService.java]: seems to be unused

File: fusionbackend/src/main/java/io/fusion/fusionbackend/rest/ecosystemmanager/AssetTypeTemplateRestService.java [hunk 2]
```diff
@@ -67,6 +72,20 @@ public AssetTypeTemplateDto getAssetTypeTemplate(@PathVariable final Long assetT
                 true);
     }
 
+    @GetMapping(path = "/assettypetemplates/{assetTypeTemplateId}/owlexport")
+    public void getAsOwlExport(@PathVariable final Long assetTypeTemplateId,
+                               HttpServletResponse response) throws IOException {
+        OntModel model = assetTypeTemplateService.getAssetTypeTemplateRdf(assetTypeTemplateId);
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/rest/ecosystemmanager/AssetTypeTemplateRestService.java:78]: Consider renaming Rdf everywhere to Owl for consistency

File: fusionbackend/src/main/java/io/fusion/fusionbackend/service/ontology/BlankNodeProcessor.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        if (blankNodes.size() > 0) {
+            insertBlankNodes(graph, blankNodes);
+        }
+        return JsonUtils.toPrettyString(root);
+    }
+
+    private static JSONObject insertBlankNodes(JSONObject node, Map<String, JSONObject> blankNodes) {
+        String id = (String) node.get("id");
+        JSONObject result = node;
+        if (id != null && id.startsWith("_:b") && node.size() == 1) {
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/service/ontology/BlankNodeProcessor.java]: maybe give string '_:b' a meaningfull variable name

**Review by jsylupp — CHANGES_REQUESTED** [review]
Your parts look good! Only three suggestions for your parts

File: fusionbackend/src/main/java/io/fusion/fusionbackend/rest/fleetmanager/FleetAssetRestService.java (hunk 1 above)
Comment (seemueller) [reply fusionbackend/src/main/java/io/fusion/fusionbackend/rest/fleetmanager/FleetAssetRestService.java]: removed

File: fusionbackend/src/main/java/io/fusion/fusionbackend/rest/ecosystemmanager/AssetTypeTemplateRestService.java (hunk 2 above)
Comment (seemueller) [reply fusionbackend/src/main/java/io/fusion/fusionbackend/rest/ecosystemmanager/AssetTypeTemplateRestService.java:78]: While RDF is not exact the same as OWL and the export as consists partly on RDFS we should difference between OWL and RDF

File: fusionbackend/src/main/java/io/fusion/fusionbackend/service/ontology/BlankNodeProcessor.java (hunk 3 above)
Comment (seemueller) [reply fusionbackend/src/main/java/io/fusion/fusionbackend/service/ontology/BlankNodeProcessor.java]: done

**Pull request merged**